docs/pages/iconography.js

```diff
@@ -0,0 +1,402 @@
+
+type PrincipleItemProps = {|
+  color: string,
+  heading: string,
+  image?: Node,
+  text: string | Node,
+|};
+
+function PrincipleItem({ color, heading, image, text }: PrincipleItemProps): Node {
+  return (
+    <Flex direction="column" gap={4}>
+      <Box
+        display="flex"
+        alignItems="center"
+        justifyContent="center"
+        height={160}
+        width={260}
+        dangerouslySetInlineStyle={{
+          __style: {
+            backgroundColor: `var(--color-${color})`,
+          },
+        }}
+      >
+        {image}
+      </Box>
+      <Heading size="400">{heading}</Heading>
+      <Box marginBottom={6}>
+        <Text>{text}</Text>
+      </Box>
+    </Flex>
+  );
+}
+
+export default function IconographyPage(): Node {
+  return (
+    <Page title="Iconography">
+      <PageHeader
+        name="Iconography"
+        description={`Our typographic system creates a content hierarchy that is scannable and efficient.
+    `}
+        type="guidelines"
+      />
+
+      <MainSection name="Principles">
+        <Flex gap={12} alignContent="between" wrap>
+          <Flex.Item flex="grow" flexBasis="0%" minWidth={275} maxWidth="45%">
+            <PrincipleItem
+              color="purple-mysticool-100"
+              image={<Obvious />}
+              heading="Obvious"
+              text={
+                <Text>
+                  Simple, intuitive, and easily recognizable. Each icon is consolidated to its
+                  minimal form, expressing fundamental aspects. Icons should feel balanced, and
+                  align elements optically.
+                </Text>
+              }
+            />
+          </Flex.Item>
+          <Flex.Item flex="grow" flexBasis="0%" minWidth={275} maxWidth="45%">
+            <PrincipleItem
+              color="green-matchacado-100"
+              image={<Intentional />}
+              heading="Intentional"
+              text={
+                <Text>
+                  Icons should be used purposefully for their intended meaning of supporting good
+                  comprehension. If questioning an icon’s use, it probably doesn’t need to be used
+                  at all.
+                </Text>
+              }
+            />
+          </Flex.Item>
+          <Flex.Item flex="grow" flexBasis="0%" minWidth={275} maxWidth="45%">
+            <PrincipleItem
+              color="blue-skycicle-100"
+              image={<Accessible />}
+              heading="Accessible"
+              text={
+                <Text>
+                  We aim to a shape that ensures readability and clearness, even at small sizes. Our
+                  icons should be understandable by a global audience of users, regardless of
+                  nationality or language.
+                </Text>
+              }
+            />
+          </Flex.Item>
+        </Flex>
+      </MainSection>
+
+      <MainSection
+        name="Typography components"
```

Comment:
Should this be "Iconography components" or "Components with Icons" maybe?

Reply:
Thanks! Fixed!